scripts/jenkins-yoctobuild-build.sh

```diff
@@ -1194,9 +1208,14 @@ if [ "$PUBLISH_ARTIFACTS" = true ]; then
         board_to_publish=qemux86-64-uefi-grub
     fi
     if is_building_board $board_to_publish; then
-        container=mender-client-qemu
-        version=$($WORKSPACE/integration/extra/release_tool.py --version-of $container --in-integration-version HEAD)
-        docker tag mendersoftware/$container:pr mendersoftware/$container:${version}
-        docker push mendersoftware/$container:${version}
+        for container in mender-client-qemu mender-client-qemu-rofs; do
+            if ! docker inspect mendersoftware/$container:pr >/dev/null; then
```

Comment:
Mmm. When is that we will be setting the version of mender-client-qemu-rofs to pr?

Reply:
We are missing in line 907 something like:

`$WORKSPACE/integration/extra/release_tool.py --set-version-of mender-client-qemu-rofs --version pr`

Reply:
Nice catch! Fixed!